roles/openshift_openstack/templates/heat_stack.yaml.j2

```diff
@@ -109,6 +109,10 @@ outputs:
     description: Id of the OpenShift API load balancer VIP port
     value: { get_attr: [api_lb, vip_port_id] }
 
+  api_lb_vip:
+    description: IP address of the OpenShift API load balancer VIP port
+    value: { get_attr: [api_lb, vip_address] }
```

Comment:
Could we reuse the `private_api_ip` here? I know that for Kuryr, a LB is required so we don't need the fallback the other output provides. But I'd prefer if we didn't define the same output value in multiple heat outputs.